Python/Tests/Analysis/AnalysisTest.cs

```diff
@@ -4301,6 +4317,34 @@ class C:
         }
 
         [TestMethod, Priority(0)]
+        public void AnnotatedAssign() {
```

Comment:
"Annotating a local variable will cause the interpreter to treat it as a local, even if it was never assigned to. Annotations for local variables will not be evaluated"
x = 1
def f():
    x: int

Type of inner x should be unknown (I suspect we fail this)